sway-error/src/error.rs

```diff
@@ -247,6 +247,12 @@ pub enum CompileError {
         span: Span,
         actually: String,
     },
+    #[error("\"{name}\" is a {actually}, not an indexable expression.")]
```

Comment:
Could you reword this to `\"{name}\" is a {actually}, which is not an indexable expression.` ?